src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  flex-shrink: 0;
+
+  &__list {
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__item {
+    position: relative;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    color: $main-text-color;
+    display: block;
+    font-weight: 500;
+    line-height: 60px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    transition: color 0.3s;
+
+    &:hover {
+      color: $third-text-color;
```

Comment:
The hover color for nav links should be `#00acdc` as specified in the task requirements. The `$third-text-color` variable is correctly set to `#00acdc`, so this should be fine.